website/search_migration/migrate.py

```diff
@@ -71,6 +71,7 @@ def migrate(delete, index=None, app=None):
         delete_old(new_index)
 
     ctx.pop()
+    populate_institutions.main(env)
```

Comment:
Are there any problems running populate_institutions multiple times on the same environment? It looks like it should handle it, but at the same time it also looks like it's going to update every node tagged with an institution, as populate_institutions is for adding or changing institution data.

If the only problem is that the institutions keys are being dropped from elastic, maybe there's a simpler way of iterating over only the institutions that we have already configured and migrating them into search, so that populate_institutions can be a separate step for its specific purpose.